### PR TITLE
Add FileMenuTools.

### PR DESCRIPTION
**Pull request by NamelessUzer**
FileMenu Tools is an application which lets you customize the context menu of Windows File Explorer.

The main features are the following:

- Add some built-in utilities that perform operations on files and folders. These utilities are explained below in more detail.
- Add custom commands which run a specific action. The possible actions are the following:
    - Run external applications
    - Copy/move to a specific folder
    - Delete specific file types
- Send the selected files as attachments to e-mail recipient(s)
- Rename the selected files using a set of rules
- Configure the "Send to..." submenu
- Enable/disable commands which are added by other applications to the context menu

homepage: https://www.lopesoft.com/index.php/en/filemenutools

### CODE REVIEW

File: bucket/filemenutools-portable.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://www.lopesoft.com/index.php/en/filemenutools",
+    "description": "FileMenu Tools is an application which lets you customize the context menu of Windows File Explorer.",
```
Comment (niheaven) [inline bucket/filemenutools-portable.json]:
```suggestion
    "description": "Application which lets you customize the context menu of Windows File Explorer.",
```

File: bucket/filemenutools-portable.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "Unregister FileMenuTools.exe",
+            "Unregister FileMenuTools"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.lopesoft.com/index.php/en/filemenutools/changelog",
+        "regex": "v([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://www.lopesoft.com/fmtools/FileMenuTools-portable.exe#/dl.7z"
```
Comment (niheaven) [inline bucket/filemenutools-portable.json]:
```suggestion
        "url": "https://www.lopesoft.com/fmtools/FileMenuTools-portable.exe#/dl.7z"
        "hash": {
            "url": "https://www.lopesoft.com/index.php/en/download/filemenu-tools",
            "regex": "(?sm)Portable.*?([a-f0-9]{32})"
        }
```

**Comment by niheaven** [comment]
You should remove `nteract` from your commit :smile:

**Comment by ssfjhh** [comment]
> You should remove `nteract` from your commit 😄

😄, How to remove nteract.json from my commit? I have googled it but I am failed. Is it a better way to close this PR and add another PR?

**Comment by niheaven** [comment]
- `git reset --soft f5a02b`
- `git rm bucket/nteract.json --cached`
- `git checkout bucket/nteract.json`
- `git commit -m "filemenutools: Add version 7.6.2"`
- `git push ... -f`

**Comment by pratikpc** [comment]
@NamelessUzer the commit currently has conflicts

Would be probably easier to create a new PR for the resolution IMHO

**Pull request closed without merging**